src/toolkit/pyroprocessing/pyre_winning.cc

```diff
@@ -0,0 +1,62 @@
+#include "pyre_winning.h"
+
+using cyclus::Material;
+using cyclus::Composition;
+using cyclus::CompMap;
+
+namespace pyro {
+
+Winning::Winning() {
+  SetCoeff();
+  current(8);
+  Rtime(1);
+  flowrate(3);
+  volume(1);
+}
+
+Winning::Winning(double new_current, 
+                 double new_time, 
+                 double new_flowrate, 
+                 double new_volume
+            ) 
+            {
+  SetCoeff();
+  current(new_current);
+  Rtime(new_time);
+  flowrate(new_flowrate);
+  volume(new_volume);
+}
+
+void Winning::SetCoeff() {
+  coul.push_back(-0.00685);
+  coul.push_back(0.20413);
+  coul.push_back(-2.273);
+  coul.push_back(11.2046);
+  coul.push_back(-19.7493);
+  ti.push_back(0.2903);
+  ti.push_back(-1.696);
+  r.push_back(0.12435);
+  r.push_back(0.7985);
```

Comment:
same here definition in the header ?

Reply:
Same as your previous comment, they are set with the vector now which is much clearer